packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -205,6 +205,35 @@ export function applyFieldOverrides(options: ApplyFieldOverrideOptions): DataFra
         context.replaceVariables,
         options.timeZone
       );
+
+      if (field.config.custom?.nested) {
+        for (const nestedFrames of field.values) {
+          for (const nf of nestedFrames) {
+            for (const valueField of nf.fields) {
+              valueField.state = {
+                scopedVars: {
+                  __dataContext: {
+                    value: {
+                      data: options.data!,
+                      frame: nf,
+                      frameIndex: index,
+                      field: valueField,
+                    },
+                  },
+                },
+              };
+
+              valueField.getLinks = getLinksSupplier(
+                nf,
+                valueField,
+                valueField.state!.scopedVars,
+                context.replaceVariables,
+                options.timeZone
+              );
+            }
+          }
+        }
+      }
```

Comment:
hm... this is not good, if we need to apply field config (and link supplier) to nested frames I think we need another type check than using custom.nested, that was only for Table panel to know about. 

This needs a new FieldType, we already have a "frame" field type. Maybe we can add a FieldType.nestedFrames?